weather_data_analyzer.py

```diff
@@ -0,0 +1,198 @@
+import csv
+import sys
+from weather_record import WeatherRecord
+from report_types \
+    import AverageTempHumidReportType, HighLowTempHumidityReportType
+
+dummy_record_for_comparison = {
+    'PKT': "2022-08-16",
+    'Mean TemperatureC': None,
+    'Max TemperatureC': -sys.maxsize,
+    'Min TemperatureC': sys.maxsize,
+    ' Min Humidity': sys.maxsize,
+    'Max Humidity': -sys.maxsize,
+    ' Mean Humidity': None
+}
```

Comment:
for comparison. To generate report 1 (highest max_temp, lowest min_temp, highest max_humidity)
Actually, It is similar to logic:
```python
min = sys.maxsize

for num in array:
       if num < min: 
             min = num

return min
```


To generate report 1, I need max, min of columns. When comparing with the file  records, i set the `dummy_record` as the first/initial `max_record/min_record` as `min = sys.maxsize` in the above code example  and then doing the comparison. 

I can set the first record from the file as `max_record/min_record`, but files data are not consistent. Means, It may possible that for a file, the first row `Max TemperatureC is empty/None`. which then gives error`Error: can't compare 'int' with None.`

Can you please tell me another way to improve that thing/logic?